cmd/controller/main.go

```diff
@@ -90,7 +90,7 @@ func main() {
 
 	mgr, err := controller.NewManager(ctx, buildCfg, cfg, manager.Options{
```

Comment:
not sure why we need to move from `cmd/manager` to `cmd/controller`. At the end of the day, what we create and start is a `manager` see [interface](https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/manager/manager.go#L51) , which is an interface that registers the multiple controllers we have, which are four. Opinions?

Reply:
I think I got a bit overzealous about renaming things 😅 

Changed it back to `cmd/manager/`